### DISCUSSION TITLE
MockV3Aggregator and price feed version test

### DISCUSSION THREAD

**Discussion by ivanGrzegorczyk** [Cyfrin/foundry-full-course-cu#1287] [General]
Hello guys, I have a question about this test on lesson 7:
```
function test_PriceFeedVersionIsAccurate() public {
     console.log("FundMe price feed version: %d", fundMe.getVersion());
     assertEq(fundMe.getVersion(), 4);
}
```
Iin order to avoid having unnecesary files in my project I decided to use the MockV3Aggregator on `@chainlink/contracts/src/v0.8/tests/MockV3Aggregator.sol` instead of adding it myself on the mocks directory. This makes the test above fail because the mock version is set to 0 and unlike the one on the couse's repo it cannot be changed. Is there a way around this? Is this test even necessary?

**Comment by ghost** [comment]
Yes, these tests are important sanity checks to know your contract is initialized correctly.

That contract it's only a mock, so you can update it in any way it suits you as long the contract keeps performing as expected, but I recommend to not modify the contract directly from the `lib` and instead create a new `mocks` folder in your project, paste the contract and change the version from `0` to `4`.

Don't forget to update the path of the import of the mock as well.
